apps/cli/src/commands/proxy.ts

```diff
@@ -250,9 +250,10 @@ const job = ({
         };
 
         ws.on("close", onClose);
-        ws.on("message", async (apduHex) => {
+        ws.on("message", async (data, isBinary) => {
           if (destroyed) return;
-
+          
+          const apduHex = isBinary ? data : data.toString();
```

Comment:
Really for understanding, `apduHex` is the hexadecimal representation of the data right ? When it's compared to "open" just below, it's implicitly converted into a string representation ?

Reply:
basically, if `isBinary` is true, what's the type of data ?

Reply:
From what the commit linked in the description is saying, here is the exact typing:

```markdown
### Event: 'message'

- `data` {Buffer|ArrayBuffer|Buffer[]}
- `isBinary` {Boolean}

Emitted when a message is received. `data` is the message content. `isBinary`
specifies whether the message is binary or not.
```

Reply:
Nice, thanks